tests/integration/test_exodus.py

```diff
@@ -42,7 +42,7 @@ def test_header_cache_control(cdn_test_url, testdata_path):
     r = requests.get(url)
     print(json.dumps(dict(r.headers), indent=2))
     assert r.status_code == 200
-    assert r.headers["cache-control"] == "max-age=600"
+    assert r.headers["cache-control"].startswith("max-age=")
```

Comment:
What do you think of doing a pattern match on the value?  e.g. verify that it matches regex `^max-age=[0-9]+$`.

The concern I have with the current approach is that it makes the test a bit too weak, e.g. if we completely failed to substitute in the config, or we had a formatting issue like `max-age="600"` rather than `max-age=600`, the test would still pass.